src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/robotics/NotifyRoboticsOnContinuousFeedHandlerTest.java

```diff
@@ -118,13 +122,31 @@ void shouldNotifyRoboticsSpecClaim_whenNoSchemaErrors() {
                                                                params.getParams().get(BEARER_TOKEN).toString()
             );
         }
+
+        @Test
+        void shouldNotNotifyRobotics_whenRpaToggleOff() {
+            // Given
+            when(featureToggleService.isRPAEmailEnabled()).thenReturn(false);
+            CaseData caseData = CaseDataBuilder.builder().atStateProceedsOfflineAdmissionOrCounterClaim().build();
+            CallbackParams params = CallbackParamsBuilder.builder().of(ABOUT_TO_SUBMIT, caseData).build();
+            boolean multiPartyScenario = isMultiPartyScenario(caseData);
+
+            // When
+            handler.handle(params);
+
+            // Then
+            verify(roboticsNotificationService, times(0)).notifyRobotics(caseData, multiPartyScenario,
```

Comment:
Minor: There is a method verifyNoInteractions(roboticsNotificationService) which could tidy this section up a bit